### ISSUE TITLE
Add ability to export constants

### ISSUE THREAD

**Issue by jgomez720** [KittyCAD/modeling-app#4716]
### Description
I am working next to @jessfraz, and mentioned that I am typing the same constants in multiple files now that I am doing more assembly work. The ability to export constants would be huge.

### KCL
**body**
```js
export height = 4
export fn body = () => {
  width = 2.5
  // Create the overall body
  chamferLength = .325
  sketch001 = startSketchOn('XZ')
    |> startProfileAt([-width / 2, height / 2], %)
      |> xLine(width, %, $chamfer1)
    |> yLine(-height, %, $chamfer2)
    |> xLine(-width, %, $chamfer3)
    |> close(%, $chamfer4)
  extrude001 = extrude(1, sketch001)
    |> chamfer({
         length = chamferLength,
         tags = [
           getNextAdjacentEdge(chamfer1),
           getNextAdjacentEdge(chamfer2),
           getNextAdjacentEdge(chamfer3),
           getNextAdjacentEdge(chamfer4)
         ]
       }, %)
  
  // Define the offset for the indentation
  offset = .125
  sketch002 = startSketchOn(extrude001, 'END')
    |> startProfileAt([
      -width/2 + offset, 
      height/2 -offset - chamferLength * sin(toRadians(45))
    ], %)
    |> angledLine({
      angle: 45,
      length: chamferLength,
    }, %)
    |> lineTo([
      width/2 - offset - chamferLength * sin(toRadians(45)), 
      height/2 - offset
    ], %)
    |> angledLine({
      angle: -45,
      length: chamferLength,
    }, %)
    |> lineTo([
      width/2 - offset, 
      -(height/2 - offset - chamferLength * sin(toRadians(45)))
    ], %)
    |> angledLine({
      angle: -135,
      length: chamferLength,
    }, %)
    |> lineTo([-(width/2 - offset - chamferLength * sin(toRadians(45))), -height/2 + offset], %)
    |> angledLine({
      angle: -225,
      length: chamferLength,
    }, %)
    |> close(%)
  extrude002 = extrude(-0.0625, sketch002)
  
  // Define constants for the screen
  screenWidth = 1.75
  screenHeight = 1
  screenYPosition = height / 2 - 0.75
  screenDepth = -.0625
  
  // Create the pocket for the screen
  sketch003 = startSketchOn(extrude002, 'start')
    |> startProfileAt([-screenWidth / 2, screenYPosition], %)
    |> xLine(screenWidth, %, $seg01)
    |> yLine(-screenHeight, %)
    |> xLine(-segLen(seg01), %)
    |> lineTo([profileStartX(%), profileStartY(%)], %)
    |> close(%)
  extrude003 = extrude(screenDepth, sketch003)
  
  // Create the speaker box
  speakerBoxWidth = 1.25
  speakerBoxHeight = 1.25
  sketch004 = startSketchOn(extrude002, 'start')
    |> startProfileAt([-1.25 / 2, -.125], %)
    |> xLine(speakerBoxWidth, %)
    |> yLine(-speakerBoxHeight, %)
    |> xLine(-speakerBoxWidth, %)
    |> close(%)
  extrude004 = extrude(-.5, sketch004)
  return extrude004
}
```

**antenna**
```js
import height from 'body.kcl'

export fn antenna (origin) {
  antennaX = origin[0]
  antennaY = origin[1]
  
  antennaBaseWidth = .5
  antennaBaseHeight = .25

  // reused constants from body

  antennaPlane = {
    plane = {
      origin = {
        x = 0,
        y = 0,
        z = height
      },
      xAxis = { x = 1, y = 0, z = 0 },
      yAxis = { x = 0, y = 1, z = 0 },
      zAxis = { x = 0, y = 0, z = 1 }
    }
  }
  
  sketch001 = startSketchOn(plane)
    |> startProfileAt([origin[0], origin[1]], %)
    |> line([antennaBaseWidth, 0], %)
    |> line([0, -antennaBaseHeight], %)
    |> line([-antennaBaseWidth, 0], %)
    |> close(%)
    
  loftPlane = offsetPlane(plane, 3)
  
  antennaTopWidth = .30
  antennaTopHeight = .05
  
  sketch002 = startSketchOn(loftPlane)
    |> startProfileAt([origin[0] + (antennaBaseWidth - antennaTopWidth)/2, origin[1] - (antennaBaseHeight - antennaTopHeight)/2], %)
    |> xLine(antennaTopWidth, %)
    |> yLine(-antennaTopHeight, %)
    |> xLine(-antennaTopWidth, %)
    |> close(%)
  
  return loft([sketch001, sketch002])
}
```

### Version
v0.29.0

### Recordings/Screenshots
![Image](https://github.com/user-attachments/assets/b54ccbcd-2358-48a8-9084-a212bf445a13)

**Comment by jtran** [comment]
Have you tried the latest nightly? I believe @nrc already implemented this in  #4677.

**Comment by jgomez720** [comment]
oh shoot. I just tried it. It worked. Super cool @nrc !!
